README.md

```diff
@@ -61,6 +61,13 @@ Special-purpose linters:
 - [dwklint](https://github.com/CVE-2008-0166/dwklint)
 - [ftfy](https://github.com/rspeer/python-ftfy)
 
+## Docker containers
+
+Docker containers are pre-built automatically and published on the Github Container Repository (GHCR). Two different release cicles are provided:
+
+- Development releases. These have a "dev.{timestamp}" tag on GHCR. These builds are automatically created upon every push to the main branch.
```

Comment:
We also need to say that the Development builds pull in the (at build time) latest commits from each of the linters, whereas the Stable builds use the latest versioned linter releases.